src/ert/ensemble_evaluator/_ensemble.py

```diff
@@ -186,12 +185,11 @@ def update_snapshot(self, events: Sequence[Event]) -> EnsembleSnapshot:
 
     async def send_event(
         self,
-        url: str,
         event: Event,
-        token: str | None = None,
         retries: int = 10,
     ) -> None:
-        async with Client(url, token) as client:
+        assert self._config is not None
+        async with Client(self._config.get_uri(), token=self._config.token) as client:
             await client.send(event_to_json(event), retries)
 
     def generate_event_creator(self) -> Callable[[Id.ENSEMBLE_TYPES], Event]:
```

Comment:
Not necessarily a part of this PR, but could we rename this to something more descriptive?